CreateDB

```diff
@@ -49,5 +54,5 @@ CREATE TABLE Contacts(
 CREATE TABLE Hashtags(
     hashtag varchar(50) not null,
```

Comment:
Missing primary key

Reply:
This is a many to many table, primary key is not needed

Reply:
Wait creo que entiendo what you mean. Aqui faltan constraints